src/components/ComboButton/ComboButton.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * @file Combo button.
+ * @copyright IBM Security 2019
+ */
+
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Fragment, useState, useRef } from 'react';
+import { ChevronDown16, ChevronUp16 } from '@carbon/icons-react';
+import { settings } from 'carbon-components';
+
+import Button from '../Button';
+import OverflowMenu from '../OverflowMenu';
+import OverflowMenuItem from '../OverflowMenuItem';
+import { TooltipDirection } from '../IconButton/IconButton';
+
+import { namespace as buttonNamespace } from '../Button/Button';
+
+import { getComponentNamespace } from '../../globals/namespace';
+
+export const namespace = getComponentNamespace('combo-button');
+
+const { prefix } = settings;
+
+const OVERFLOW_MENU_ICON = {
+  OPEN: ChevronUp16,
+  CLOSE: ChevronDown16,
+};
+
+const ComboButton = props => {
+  const [overflowMenuIcon, setOverflowMenuIcon] = useState(
```

Comment:
It might be better to toggle a boolean state value instead of a component object because I would think that objects would have to be serialized to be compared. You can then use the boolean value to determine which icon to render.